pkg/docker/registry.go

```diff
@@ -3,10 +3,21 @@ package docker
 import (
 	"context"
 	"runtime"
+	"strings"
 
 	"github.com/aws/eks-anywhere/pkg/logger"
 )
 
+// These constants are temporary since currently there is a limitation on harbor
+// Harbor requires root level projects but curated packages private account currently
+// doesn't have support for root level
+const (
+	packageProdLocation = "783794618700.dkr.ecr.us-west-2.amazonaws.com"
+	packageDevLocation  = "857151390494.dkr.ecr.us-west-2.amazonaws.com"
+	publicProdECRName   = "eks-anywhere"
+	publicDevECRName    = "l0g8r8j6"
```

Comment:
could we just use the same "folder" for both prod and dev? I would vote for `eks-anywhere`

Reply:
This will not be ideal because for dev, the controller will be using l0gr... to fetch images, and if we make this change, it will break the pull of images.